include/evm_runtime/tables.hpp

```diff
@@ -346,8 +346,9 @@ struct [[eosio::table]] [[eosio::contract("evm_contract")]] config
     binary_extension<consensus_parameter_type> consensus_parameter;
     binary_extension<eosio::name> token_contract; // <- default(unset) means eosio.token
     binary_extension<uint32_t> queue_front_block;
+    binary_extension<uint64_t> ingress_gas_limit = 21000;
```

Comment:
For consistency we should put all the `binary_extension` default values in `config_wrapper` constructor as we are doing for the other fields